### ISSUE TITLE
Difference between two prediction results(ESP_prediction provided on Github and online server)

### ISSUE THREAD

**Issue by Liujing19830425** [AlexanderKroll/ESP#13]
Dear Alex:
I found that there are differences in the prediction results using two methods (ESP_prediction provided on Github and online server), some of which are quite significant. Is the reason for this difference due to the different models? Which is more reliable in this situation? About ESP_prediciton, can you provide the latest trained model?

Looking forward to your reply!
Jing

**Comment by maxall41** [comment]
Not the author but I think the website uses the production model which includes the test set in the training data. In the training notebook there is a production model and a validation/testing model which one are you using?

**Comment by Liujing19830425** [comment]
> Not the author, but I think the website uses the production model, which includes the test set in the training data. In the training notebook, there is a production model and a few different models for testing different descriptors which one are you using? Also the model weights for the production model can be found here: https://github.com/AlexanderKroll/ESP/blob/main/data/model_weights/xgboost_model_production_mode.dat

Dear Alex:
First of all, thank you for your reply！
"xgboost_model_production_mode_gnn_esm1b_ts.dat" is used in the ESP_prediction program, and is also mentioned in the paper to improve the effect.The results of the test based on this model are different from the website results.So I wonder what the reasons for this difference are.


Looking forward to your reply!
Jing

**Comment by maxall41** [comment]
I am not the author of the paper. It is interesting that `xgboost_model_production_mode_gnn_esm1b_ts.dat` provides different results than the web server. Maybe @AlexanderKroll knows why this is happening?

**Comment by AlexanderKroll** [comment]
Hi, sorry for my late response. As you can read on the homepage of the webserver, the predictions on the webserver are made with our new ProSmith model that achieves superior prediction results.

**Comment by Liujing19830425** [comment]
> Hi, sorry for my late response. As you can read on the homepage of the webserver, the predictions on the webserver are made with our new ProSmith model that achieves superior prediction results.

**Comment by Liujing19830425** [comment]
> Hi, sorry for my late response. As you can read on the homepage of the webserver, the predictions on the webserver are made with our new ProSmith model that achieves superior prediction results.
Dear Alex:
First of all, thank you for your reply！
Is it convenient to provide this ProSmith model to us?

Thanks，
Looking forward to your reply!
Jing

**Comment by Liujing19830425** [comment]
> Hi, sorry for my late response. As you can read on the homepage of the webserver, the predictions on the webserver are made with our new ProSmith model that achieves superior prediction results.

**Comment by Liujing19830425** [comment]
Dear Alex:
First of all, thank you for your reply！

Is it convenient to provide this new ProSmith model to us?

Thanks,
Jing

**Comment by AlexanderKroll** [comment]
Dear Jing,

you can find the new ProSmith model for the enzyme-substrate prediction task here: https://github.com/AlexanderKroll/ESP_ProSmith

**Comment by Liujing19830425** [comment]
> Dear Jing,
> 
> you can find the new ProSmith model for the enzyme-substrate prediction task here: https://github.com/AlexanderKroll/ESP_ProSmith

Thanks
